packages/sdk/package.json

```diff
@@ -22,6 +22,20 @@
     "./dist/index.cjs.js": "./dist/index.browser.cjs.js",
     "./dist/index.esm.js": "./dist/index.browser.esm.js"
   },
+  "browserslist": {
+    "production": [
```

Comment:
Seems...unrelated? Why the browserlist changes for this?

Reply:
viem uses bigint exponentation. targeting lower browser versions polyfills with math.pow(), which doesn't work